dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -877,6 +906,160 @@ public void download(final List<String> args) {
         }
     }
 
+
+    /**
+     * Creates a WES API object and sets the endpoint.
+     *
+     * @param wesUrl URL to WES endpoint
+     */
+    public WorkflowExecutionServiceApi getWorkflowExecutionServiceApi(String wesUrl) {
+        WorkflowExecutionServiceApi clientWorkflowExecutionServiceApi = new WorkflowExecutionServiceApi();
+        ApiClient wesApiClient = clientWorkflowExecutionServiceApi.getApiClient();
+
+        INIConfiguration config = Utilities.parseConfig(this.getConfigFile());
+        SubnodeConfiguration configSubNode = null;
+        try {
+            configSubNode = config.getSection("WES");
+        } catch (Exception e) {
+            //?????what should the return code be?????
+            exceptionMessage(e, "Could not get WES section", ENTRY_NOT_FOUND);
```

Comment:
This may get slightly tricky... If you add command-line options to allow the url and auth to be specified on the command-line, then the section should not be required and you should not throw an error here.